test/testdrive-old-kafka-src-syntax/pr-24663-regression.td

```diff
@@ -123,7 +125,7 @@ $ kafka-ingest format=avro topic=dbz-no-before key-format=avro key-schema=${keys
   WHERE s.name IN ('dbz_no_before')
   GROUP BY s.name
   ORDER BY s.name
-true true 0
+true false 0
```

Comment:
@def- I wanted to make this dependent of the configuration of `storage_use_continual_feedback_upsert`, but didn't get it to work, `set-from-sql ...` only runs as normal user, but I need to run as `mz_system` do do a `SHOW storage_use_continual_feedback_upsert`.

Would really have been ideal to have a var like `leaves-tombstones`... :see_no_evil:

Reply:
we can pass variables in from mzcompose via Testdrive's `--var=storage_use_continual_feedback_upsert=false/true` and then use `${arg.storage_use_continual_feedback_upsert}` in the .td files. Will push a fix later today